mlflow/utils/virtualenv.py

```diff
@@ -245,7 +287,16 @@ def _get_or_create_virtualenv(local_model_path, env_id=None):
         raise
 
 
-def _execute_in_virtualenv(activate_cmd, command, install_mlflow, command_env=None):
+def _execute_in_virtualenv(
+    activate_cmd,
+    command,
+    install_mlflow,
+    command_env=None,
+    synchronous=True,
+    capture_output=False,
+    env_root_dir=None,
```

Comment:
Do we need this argument? Can we include `env_root_dir` in `command_env` when we call this function?

Reply:
I think `env_root_dir` is better as a standalone parameter because it's well-documented and corresponds to the `PyFuncBackend` API.

Reply:
Agree!